staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/types.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fairqueuing
+
+import (
+	"time"
+)
+
+// Packet is a temporary container for "requests" with additional tracking fields
+// required for the functionality FQScheduler
+type Packet struct {
```

Comment:
We are not doing networking, perhaps this should be called something more like "Request"?

Reply:
updated name to Request